src/Service/PsCountryService.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\PrestaShop\Service;
+
+use Country;
+use Country as PsCountry;
+use MyParcelNL\Pdk\Base\Service\CountryCodes;
+use MyParcelNL\Pdk\Facade\Pdk;
+use MyParcelNL\Pdk\Facade\Platform;
+use MyParcelNL\PrestaShop\Contract\PsCountryServiceInterface;
+use MyParcelNL\Sdk\src\Support\Str;
+
+/**
+ * @template T of PsCountry
+ * @extends \MyParcelNL\PrestaShop\Service\PsSpecificObjectModelService<T>
+ */
+final class PsCountryService extends PsSpecificObjectModelService implements PsCountryServiceInterface
+{
+    /**
+     * @var array<string, int|false>
+     */
+    private static array $countryIdIsoCache = [];
+
+    /**
+     * @TODO: Remove this when Carrier Capabilities service is implemented.
+     *
+     * @param  string $carrierName
+     *
+     * @return array
+     */
+    public function getCountriesForCarrier(string $carrierName): array
+    {
+        // Resolve carrier identifier
+        [$resolvedCarrierName] = explode(':', $carrierName);
+
+        $platform            = Platform::getPlatform();
+        $allCarrierCountries = Pdk::get('countriesPerPlatformAndCarrier')[$platform] ?? [];
+        $countriesForCarrier = $allCarrierCountries[$resolvedCarrierName] ?? [];
+
+        if (true === ($countriesForCarrier['fakeDelivery'] ?? null)) {
```

Comment:
Waarom moet je kijken of dit een fakeDelivery is? Wanneer is iets een fakeDelivery?

Reply:
is uit de data die gekopieerd is uit de delivery options, ik wilde dat expres niet aanpassen om zo het _tijdelijke_ onderhoud te verminderen